src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -482,6 +488,235 @@ describe('526 helpers', () => {
     });
   });
 });
+describe('transform', () => {
+  const transformedMinimalData = {
+    form526: {
+      hasTrainingPay: false,
+      isVAEmployee: false,
+      homelessOrAtRisk: 'no',
+      phoneEmailCard: {
+        primaryPhone: '1231231234',
+        emailAddress: 'asdf@as.asdf',
+      },
+      mailingAddress: {
+        country: 'USA',
+        addressLine1: 'asdf',
+        city: 'asdf',
+        state: 'AL',
+        zipCode: '19234',
+      },
+      ratedDisabilities: [
+        {
+          name: 'First Condition',
+          ratedDisabilityId: '0',
+          ratingDecisionId: '63655',
+          diagnosticCode: 5238,
+          decisionCode: 'SVCCONNCTED',
+          decisionText: 'Service Connected',
+          ratingPercentage: 100,
+          disabilityActionType: 'INCREASE',
+        },
+      ],
+      serviceInformation: {
+        servicePeriods: [
+          {
+            serviceBranch: 'Air Force',
+            dateRange: {
+              from: '1900-01-01',
+              to: '1901-01-01',
+            },
+          },
+        ],
+      },
+      standardClaim: false,
+    },
+  };
+
+  const transformedMaximalData = {
```

Comment:
This file is already getting pretty long. What do you think about putting these expected outputs into their own file? It'd save 220+ LoC in here.

Reply:
Fair enough.